tests/test_bolt2-30-channel_type-open-accept-tlvs.py

```diff
@@ -107,9 +104,11 @@ def test_open_channel_empty_type(runner: Runner, with_proposal: Any) -> None:
         Block(blockheight=102, txs=[tx_spendable]),
         Connect(connprivkey="02"),
         ExpectMsg("init"),
-        TryAll(
-            # | 12/13 | `option_static_remotekey`        | Static key for remote output
-            Msg("init", globalfeatures="", features=bitfield(13))
+        # | 12/13 | `option_static_remotekey`        | Static key for remote output
+        Msg(
+            "init",
+            globalfeatures=runner.runner_features(globals=True),
+            features=runner.runner_features(additional_features=[13]),
```

Comment:
same consideration here

Reply:
Here I'm not so sure. I did add 13 as an additional field and 13 was the bit that the test used previously. Does this change the meaning of the test as well?